### DISCUSSION TITLE
Does this `check-success~=^deploy-` check everything that starts with deploy?

### DISCUSSION THREAD

**Discussion by DrAugus** [Mergifyio/mergify#5119] [Q&A] [answered]
![1722837562675](https://github.com/user-attachments/assets/8fbd4e9c-3dc5-44e8-bbed-4934d2d3dbdc)
![image](https://github.com/user-attachments/assets/c1fcbfc9-2508-4146-a61e-53ea51c2cd7a)
here is my [mergify](https://github.com/DrAugus/draugus.github.io/blob/master/.github/mergify.yml)

**Comment by jd** [comment, marked as answer]
👋 
`check-success~=^deploy-` is true if _any_ check starting with `deploy-` is success.
There is no way to know every CI that will report to GitHub (this is not the way GitHub API works). You therefore **need** to list every CI that you expect.
